### ISSUE TITLE
expose client IP address on server side in nodejs

### ISSUE THREAD

**Issue by blerest** [grpc/grpc#8740]
When the nodejs server receives an RPC call, is it possible to know the client IP?

**Comment by blerest** [comment]
`getPeer()`  on the context of the request returns the IP as a string (`ipv4:89.159.8.233:54103`). Does it possible to have an object instead of a URI string (#5693) ?

```
{
    protocol: 'ipv4',
    ip: '89.159.8.233',
    port: 54103
}
```

**Comment by kazimanzurrashid** [comment]
Any update on this?

**Comment by lesomnus** [comment]
I think this proposal has been accepted and implemented; I use it on the current version(4.0.2).
